lib/far_mar/base.rb

```diff
@@ -0,0 +1,18 @@
+class FarMar::Base
+
+  def initialize (attrs)
+
+  end
+
+  def read_file
+    "class needs to be defined in order to read file"
```

Comment:
We've never talked about this, but just a thought. Putting a default method here is a great idea, and if a class existed without the method this file would run, then most likely cause an error down the road because a string would be returned instead of an array. So the thought would be to instead of eventually having an error, just immidietely cause an error with a relevant error message. 

```rb
def read_file
   raise StandardError, "#{this.class} needs to define `.read_file`"
end
```